src/Bicep.Core/PrettyPrint/Options/NewLineOption.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+namespace Bicep.Core.PrettyPrint.Options
+{
+    public enum NewlineOption
+    {
+        Auto,
+        LF,
+        CRLF,
+        CR

```

Comment:
Is CR regarded a newline character on its own?

Reply:
Yup. Mac systems (pre-OS X) uses CR as newline character.

Reply:
And we actually support it in bicep although it's not explicitly tested.